src/main/java/uk/gov/hmcts/reform/civil/service/docmosis/claimantresponse/InterlocutoryJudgementDocGenerator.java

```diff
@@ -0,0 +1,49 @@
+package uk.gov.hmcts.reform.civil.service.docmosis.claimantresponse;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.civil.documentmanagement.DocumentManagementService;
+import uk.gov.hmcts.reform.civil.documentmanagement.model.CaseDocument;
+import uk.gov.hmcts.reform.civil.documentmanagement.model.DocumentType;
+import uk.gov.hmcts.reform.civil.documentmanagement.model.PDF;
+import uk.gov.hmcts.reform.civil.model.CaseData;
+import uk.gov.hmcts.reform.civil.model.docmosis.DocmosisDocument;
+import uk.gov.hmcts.reform.civil.model.docmosis.InterlocutoryJudgementDoc;
+import uk.gov.hmcts.reform.civil.model.docmosis.InterlocutoryJudgementDocMapper;
+import uk.gov.hmcts.reform.civil.service.docmosis.DocumentGeneratorService;
+import uk.gov.hmcts.reform.civil.service.docmosis.TemplateDataGenerator;
+
+import static uk.gov.hmcts.reform.civil.service.docmosis.DocmosisTemplates.INTERLOCUTORY_JUDGEMENT_DOCUMENT;
+
+@Service
+@Getter
+@RequiredArgsConstructor
+public class InterlocutoryJudgementDocGenerator implements TemplateDataGenerator<InterlocutoryJudgementDoc> {
+
+    private final InterlocutoryJudgementDocMapper mapper;
+    private final DocumentManagementService documentManagementService;
+    private final DocumentGeneratorService documentGeneratorService;
+
+    public CaseDocument generateInterlocutoryJudgementDoc(CaseData caseData, String authorisation) {
+
```

Comment:
please remove empty lines like that everywhere you added them

Reply:
Removed empty line